x-pack/plugins/code/server/ctags/ctags_reader.test.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import sinon from 'sinon';
+
+import { CtagsReader } from './ctags_reader';
+import { ConsoleLoggerFactory } from '../utils/console_logger_factory';
+
+afterEach(() => {
+  sinon.restore();
+});
+
+test('Read ctags for c', () => {
+  const ctagsReader = new CtagsReader(new ConsoleLoggerFactory());
+  ctagsReader.readLine('foo\tsample.c\t/^int foo(int a, int b) {$/;\"\tfunction\tline:8\tsignature:(int a, int b)');
```

Comment:
I want to confirm whether the ctags handle the files with same name. Like there two 'same' local symbols located at two different `sample.c` under different folders. Does ctags generate same tag description for them?

Reply:
Yes, it just uses `name` which isn't as accurate as `qname`

Reply:
Hmm, I see.